File: src/test/java/ru/ezhov/rocket/action/configuration/ui/ReflectionRocketActionConfigurationRepositoryTest.java
package ru.ezhov.rocket.action.configuration.ui;

import org.junit.Test;
import ru.ezhov.rocket.action.api.RocketActionConfiguration;
import ru.ezhov.rocket.action.infrastructure.ReflectionRocketActionConfigurationRepository;

import java.util.List;

import static org.junit.Assert.assertFalse;

public class ReflectionRocketActionConfigurationRepositoryTest {

    @Test
    public void shouldCreateConfigurationList() {
        ReflectionRocketActionConfigurationRepository repository = new ReflectionRocketActionConfigurationRepository();

        final List<RocketActionConfiguration> all = repository.all();

        assertFalse(all.isEmpty());
    }

}

File: src/test/kotlin/ru/ezhov/rocket/action/configuration/ui/ReflectionRocketActionConfigurationRepositoryTest.kt
package ru.ezhov.rocket.action.configuration.ui

import org.junit.Assert
import org.junit.Test
import ru.ezhov.rocket.action.configuration.infrastructure.ReflectionRocketActionConfigurationRepository

class ReflectionRocketActionConfigurationRepositoryTest {
    @Test
    fun shouldCreateConfigurationList() {
        val repository = ReflectionRocketActionConfigurationRepository()
        repository.load()
        val all = repository.all()
        Assert.assertFalse(all.isEmpty())
    }
}